src/components/dac_dataset_table/DACDatasetTableCellData.js

```diff
@@ -0,0 +1,143 @@
+import {button, div, h} from 'react-hyperscript-helpers';
+import {Link} from 'react-router-dom';
+import {isNil} from 'lodash/fp';
+import {styles} from './DACDatasetTable';
+import { findPropertyValue, getDataUseCodes } from '../../utils/DatasetUtils';
+import {span} from 'react-hyperscript-helpers';
+import {DAC} from '../../libs/ajax';
+
+export const consoleTypes = {
+  CHAIR: 'chair',
+  DATA_SUBMITTER: 'dataSubmitter',
+};
+
+export function duosIdCellData({dataset, label='duosIdCellData'}) {
+  return {
+    data: div({
+      style: {
+        display: 'flex',
+        alignItems: 'center',
+      }
+    }, [h(Link, {to: `dataset_statistics/${dataset.dataSetId}`}, [dataset.alias])]),
+    value: dataset.alias,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.duosId },
+    label
+  };
+}
+
+export function dataSubmitterCellData({dataset, label='dataSubmitterCellData'}) {
+  const dataSubmitter = findPropertyValue(dataset, 'Data Depositor');
+  return {
+    data: dataSubmitter,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.dataSubmitter },
+    label
+  };
+}
+
+export function datasetNameCellData({dataset, label='datasetNameCellData'}) {
+  const datasetName = findPropertyValue(dataset, 'Dataset Name');
+  return {
+    data: div({
+      style: {
+        display: 'flex',
+        alignItems: 'center',
+      }
+    }, [h(Link, {to: `dataset_registration/${dataset.dataSetId}`}, [datasetName])]),
+    value: datasetName,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.datasetName },
+    label
+  };
+}
+
+export function dataCustodianCellData({dataset, label='dataCustodianCellData'}) {
+  const dataCustodian = findPropertyValue(dataset, 'Data Depositor');
+  return {
+    data: dataCustodian,
+    id: dataset.dataSetId,
+    cellStyle: { width: styles.cellWidths.dataCustodian },
+    label
+  };
+}
+
+export function dataUseCellData({dataset, label='dataUseCellData'}) {
+  getDataUseCodes(dataset);
```

Comment:
You probably want to lazily enrich the getDataUseCodes based on the record that are visible because this can lead to many async calls, depending on the dataset configuration.  The behavior that was accepted on the Researcher Dataset table was users won't be able to search all of the values until every row is visible, but they won't have to wait for all of the calls to complete upfront.  @rushtong and I have talked about improving this so that the backend object might include the necessary information without the frontend making all of the calls.